### PR TITLE
Product addition: Opera

### PR DESCRIPTION
**Pull request by Quoot**
**Type of pull request:** product addition

---

This is a pull request to add Opera as a product. Only note would be that I couldn't figure out how to run it locally to test if everything at least _functioned_, as I am rather new at GitHub, but I double checked all of the additions and everything looks like it should be good format-wise.

### CODE REVIEW

File: products/opera.toml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+[rubric.security]
+value = "no"
+notes = [
+  "Neither the Privacy Policy nor the Security Policy outlined any of their own practices regarding security of user data."
+]
+
+[rubric.third-party-collection]
+value = "no"
+notes = [
+  "While they do not collect data from third parties, they often use thrid parties to interact with your data, such as personalized ads and news.\r\n\r\nA full list of third parties they use for this purpose is listed on their Privacy Statement"
```
Comment (milesmcc) [inline products/opera.toml:26]: I think this sentence needs a comma at the end.

File: products/opera.toml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+[rubric.third-party-access]
+value = "yes-specified-noncritical"
+citations = [
+  "Our applications and services include third-party technology or code, some of which may use your data in different ways. When such third-party technologies use previously collected data, they typically act as data processors for us. When they collect data on their own, they typically act as independent data controllers."
+]
+
+[rubric.data-collection-reasoning]
+value = "somewhat"
+notes = [
+  "They only mention why advertisement data is collected."
```
Comment (milesmcc) [inline products/opera.toml:56]: Is it possible to add a citation here?

**Review by milesmcc — CHANGES_REQUESTED** [review]
Thanks, Quintin! This looks great. I just left a few very small comments, then we'll be all set to merge!

File: products/opera.toml (hunk 1 above)
Comment (Quoot) [reply products/opera.toml]: Done.

File: products/opera.toml (hunk 2 above)
Comment (Quoot) [reply products/opera.toml:59]: I added a citation here but I wasn't too sure how to format it well. Right now I have this:

"We collect this data pursuant to our legitimate business interests. As noted in our End User License Agreements and Terms of Service, our applications are ad-supported. Collecting this information helps us offset the costs of our business and helps provide a more personalized experience for the user."

Does my note make it clear enough that they're talking about ad data?

**Review by milesmcc — APPROVED** [review]

**Comment by milesmcc** [comment]
Looks good to me! Thanks for writing this out.

**Pull request merged**